pkg/doc/vc/crypto/crypto.go

```diff
@@ -315,6 +344,63 @@ func (c *Crypto) getSigner(did, didKeyType, didPrivateKey, creator string, opts
 	}
 }
 
+func validateProofPurpose(proofPurpose, method string, vdri vdriapi.Registry) error {
+	// TODO only trustbloc sidetree supports creating different blocks throuh uniregistrar.
+	if !strings.Contains(method, "did:trustbloc") {
```

Comment:
How do you know if this DID is from uniregistrar? someone might have passed private key and creator.

Reply:
Well, may be I need to update comment. I meant test only for trustbloc DIDs.